### PR TITLE
fix source bug

### LINKED ISSUE #3: [BUG]: Source is showing `undefined` 🐛
**Describe the bug**
In the paragraph info section, the source is showing `undefined`

**Expected behavior**
There should be a link of the source with the link as a text, e.g. [https://github.com/TechStarHub/svelte-typer](https://github.com/TechStarHub/svelte-typer)

**Screenshots**
image

**Desktop (please complete the following information):**
Not Necessary 

**Smartphone (please complete the following information):**
Not Necessary 

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by spudoodle**
#### Reference Issues / PRs

closes #3

#### What does this implement/fix?

fixes source is showing undefined bug

source bug fix

#### Any other comments?

No

### CODE REVIEW

**Comment by iamBijoyKar** [comment]
Hey, @Achal-00 nice work, but can you make link like this 

Source: [https://americanliterature.com/author/stephen-leacock/short-story/the-new-food](https://americanliterature.com/author/stephen-leacock/short-story/the-new-food)

The text should be a link and the link text should be same as the link `href`

**Comment by spudoodle** [comment]
Hey @iamBijoyKar, I have implemented the requested functionality.

File: src/components/Editor.svelte [hunk 1]
```diff
@@ -203,7 +203,7 @@
             <div class="flex flex-col gap-1 mt-2">
                 <h4 class="font-medium">Title: {paraData.title}</h4>
                 <h4 class="font-medium">Author: {paraData.author}</h4>
-                <h4 class="font-medium">Source: {paraData.source}</h4>
+                <h4 class="font-medium">Source: <a href={paraData.resource} target="_blank">{paraData.resource}</a></h4>
```
Comment (iamBijoyKar) [inline src/components/Editor.svelte]: make link blue to make it obvious and give it a underline on hover

**Review by iamBijoyKar — CHANGES_REQUESTED** [review]

File: src/components/Editor.svelte (hunk 1 above)
Comment (iamBijoyKar) [reply src/components/Editor.svelte]: use these classes `hover:underline text-blue-800`

Comment (spudoodle) [reply src/components/Editor.svelte]: Done!

**Review by iamBijoyKar — APPROVED** [review]
👍

**Comment by iamBijoyKar** [comment]
Nice work @Achal-00 👍, looking forward to your next PR 😄

**Pull request merged**